app/auth/keycloak_util.py

```diff
@@ -63,15 +63,17 @@ def get_user_info(self, user_id: str) -> KCInfo:
         try:
             kadm = self.keycloak_admin
             user = kadm.get_user(user_id)
-            iam_roles = [role["name"] for role in kadm.get_realm_roles_of_user(user_id)]
+            iam_roles = [
+                role["name"] for role in kadm.get_composite_realm_roles_of_user(user_id)
```

Comment:
The get_realm_roles_of_user method does not retrieve all the roles. 

get_composite_realm_roles_of_user retrieves the same roles as with the OIDC authorization.